### PR TITLE
C#: Convert flow summaries to CSV format.

### PR DESCRIPTION
**Pull request by michaelnebel**
The following is covered by this PR

- System.String
- System.Text.StringBuilder
- System.Lazy
- System.Nullable
- System.Convert

### CODE REVIEW

File: csharp/ql/lib/semmle/code/csharp/frameworks/system/Text.qll [hunk 1]
```diff
@@ -24,6 +25,97 @@ class SystemTextStringBuilderClass extends SystemTextClass {
   Method getAppendFormatMethod() { result = this.getAMethod("AppendFormat") }
 }
 
+/** Data flow for `System.Text.StringBuilder`. */
+private class SystemTextStringBuilderFlowModelCsv extends SummaryModelCsv {
```
Comment (hvitved) [inline csharp/ql/lib/semmle/code/csharp/frameworks/system/Text.qll:29]: Add a `private import semmle.code.csharp.frameworks.System` to the module `Frameworks` in `ExternalFlow.qll` to ensure bidirectional importing.

**Review by hvitved — CHANGES_REQUESTED** [review]
LGTM, just one comment.

File: csharp/ql/lib/semmle/code/csharp/frameworks/system/Text.qll (hunk 1 above)
Comment (hvitved) [reply csharp/ql/lib/semmle/code/csharp/frameworks/system/Text.qll:29]: (Comment above updated)

Comment (michaelnebel) [reply csharp/ql/lib/semmle/code/csharp/frameworks/system/Text.qll:29]: Will fix, but why is it needed?

**Review by hvitved — APPROVED** [review]
LGTM. But let's not merge before we see the result of the DCA run.

File: csharp/ql/lib/semmle/code/csharp/frameworks/system/Text.qll (hunk 1 above)
Comment (hvitved) [reply csharp/ql/lib/semmle/code/csharp/frameworks/system/Text.qll:29]: The flow summaries are compiled down to data flow nodes and atomic data-flow steps, which is a cached computation, so we want to make sure that this cached computation knows all flow summaries. If not, we may end up having to recompute this expensive computation.

**Pull request merged**